src/main.js

```diff
@@ -1,28 +1,33 @@
 import Vue from 'vue'
 import Buefy from 'buefy'
+import 'buefy/dist/buefy.css'
```

Comment:
Try importing specific components from Buefy instead of the whole bundle, similar to the fa icons.

Reply:
I've tried: 

1. `import { Tabs, <etc.> } from 'buefy'`
which didn't reduce the bundle size at all.

2. `import { Tabs } from 'buefy/src/components/Tabs'
(like with the icons): the site wouldn't load at all.

Reply:
Ah, well shoot. Nevermind :)

Reply:
Well, there's definitely something I'm doing wrong, because buefy site says that the size should be small 

> Even though this is gonna import ALL components to your application, Buefy full bundle is optimized and still pretty small:
> script is 32KB min+gzip
> style is 28KB min+gzip (with bulma included)

And it's not what I see when I build the site.